src/components/Greeting.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreeting } from '../store/greetings';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const greet = useSelector((state) => state.greeting);
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  });
+
+  const handleGreeting = () => {
+    dispatch(fetchGreeting());
+  };
+
+  return (
+    <div className="div">
+      <h1>
+        Greetings:
+        {greet}
+      </h1>
+      <button onClick={handleGreeting} type="submit">Greet Randomly</button>
+    </div>
```

Comment:
- [ ] I think there is an error here in your front end project so kindly make sure to solve it since I can't see any greetings in the page even after pressing on the button I couldn't see the greetings so kindly check again and figur out the issure and work on it to make your Project works properly🚀💯💯